packages/select/src/FilterMultiSelect/components/MultiSelectOption/MultiSelectOption.tsx

```diff
@@ -1,22 +1,25 @@
-import React, { useMemo } from "react"
+import React, { useMemo, HTMLAttributes } from "react"
 import { useFocusRing } from "@react-aria/focus"
 import { useOption } from "@react-aria/listbox"
 import { mergeProps } from "@react-aria/utils"
 import classnames from "classnames"
 import { v4 } from "uuid"
 import { VisuallyHidden } from "@kaizen/a11y"
+import { OverrideClassName } from "@kaizen/component-base"
 import { Icon } from "@kaizen/component-library"
 import check from "@kaizen/component-library/icons/check.icon.svg"
 import { Badge } from "@kaizen/draft-badge"
 import { MultiSelectItem } from "../../../types"
 import { useSelectionContext } from "../../provider"
 import styles from "./MultiSelectOption.module.scss"
 
-export interface MultiSelectOptionProps {
+export interface MultiSelectOptionProps
+  extends OverrideClassName<HTMLAttributes<HTMLSpanElement>> {
```

Comment:
You've also added native attributes to the props API but it's not implemented in the component. Let's just remove that part because it's going to add complications to this PR.

Reply:
Thanks @dougmacknz , that's done now.